utils.js

```diff
@@ -7222,11 +7222,16 @@ export const addEvidenceForImports = (pkgList, allImports) => {
         ? [name, `${group}/${name}`, `@${group}/${name}`]
         : [name];
     for (const alias of aliases) {
-      if (impPkgs.includes(alias)) {
-        const evidences = allImports[alias];
-        if (evidences) {
-          pkg.scope = "required";
-          let importedModules = new Set();
+      const all_includes = impPkgs.filter(
+        (find_pkg) =>
+          find_pkg.startsWith(alias) &&
+          (find_pkg.length === alias.length || find_pkg[alias.length] === "/")
+      );
+      if (impPkgs.includes(alias) || all_includes.length) {
+        let importedModules = new Set();
+        pkg.scope = "required";
+        for (const subevidence of all_includes) {
```

Comment:
Wondering if we there is a way to avoid this nested loop. For large boms with 1000s of components this might be slow. wdyt?

Reply:
To shorten the execution time, I think it is possible to make a condition under which, if we do not find direct imports in required dependencies, then only then try to look for sub-dependencies.
But I also think that displaying in occurrences all imports (even those where there were direct ones), and not just direct ones, can be useful.